client/my-sites/stats/stats-views/style.scss

```diff
@@ -49,6 +49,7 @@
 
 	&.level-3 {
 		background-color: var( --color-primary-light );
+		color: white;
```

Comment:
let's use `var( --color-white )` to make sure we can switch it for different color themes

Reply:
Should the same be done [here](https://github.com/Automattic/wp-calypso/blob/5384b0e48a082ce46f249a1510982fc054cb47b0/client/my-sites/stats/stats-views/style.scss#L57) and [here](https://github.com/Automattic/wp-calypso/blob/5384b0e48a082ce46f249a1510982fc054cb47b0/client/my-sites/stats/stats-views/style.scss#L62) as well?

Reply:
Good point, yes! 👍

Reply:
Addressed! :+1:

Reply:
@torres126 yup! thanks!